.github/workflows/performancetest.yml

```diff
@@ -0,0 +1,40 @@
+name: Performance-Test
+
+on:  
+  push:
+    branches:
+      - development
+
+jobs:
+  slsart-perf:
+    name: slsart Performance Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    env:
+      NPM_CONFIG_PREFIX: "~/.npm-global"
+      STAGE_NAME: merge-perf-test
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - run: npm install -g serverless@1.38.0
+      - run: npm install -g serverless-artillery
+      # once a folder with slsart script.yml and serverless.yml are created and in the repo
+      # we can copy them into the working dir to run those specific tests with those configurations
+      # - currently this just runs the defaults
+      # examples for future:
+      # - run: cp ./slsart/script.yml ./
+      # - run: cp ./slsart/serverless.yml ./
+      - run: ~/.npm-global/bin/slsart deploy --stage $STAGE_NAME
+      - run: ~/.npm-global/bin/slsart invoke --stage $STAGE_NAME
+      - run: ~/.npm-global/bin/slsart remove --stage $STAGE_NAME
```

Comment:
What do the above three commands do? Maybe some comments wouldn't go amiss?

Reply:
These are the basic serverless-artillery commands to deploy the functions to aws, invoke them, and then take them down. I figured if someone was working within the script, they'd be familiar with slsart, so I didn't want to add extra comments.